iree/modules/check/dialect/CMakeLists.txt

```diff
@@ -12,8 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# bazel_to_cmake: DO NOT EDIT
 add_subdirectory(test)
 
+# Unimplemented filegroup td_files
```

Comment:
should be safe to ignore and remove this comment

Reply:
Removed in both files.